administrator/components/com_config/model/form/application.xml

```diff
@@ -6,33 +6,33 @@
 		<field
 			name="caching"
 			type="list"
-			default="2"
 			label="COM_CONFIG_FIELD_CACHE_LABEL"
 			description="COM_CONFIG_FIELD_CACHE_DESC"
+			default="2"
 			filter="integer"
 			>
 			<option value="0">COM_CONFIG_FIELD_VALUE_CACHE_OFF</option>
-			<option value="1">COM_CONFIG_FIELD_VALUE_CACHE_CONSERVATIVE</option>
```

Comment:
fixed in https://github.com/zero-24/joomla-cms/pull/17/commits/f74f7ebf6ea29666808bd006c87038fdb722004f